src/schema/schema_create.c

```diff
@@ -540,7 +605,14 @@ __create_colgroup(WT_SESSION_IMPL *session, const char *name, bool exclusive, co
         WT_ERR(__wt_schema_open_colgroups(session, table));
     }
 
+    if (table->is_tiered_shared && active) {
```

Comment:
This needs a comment. It is sending us back to the beginning of the function. It would be good if we could figure out a way to not have those goto's for active/not-active in various functions, but just do both parts together or wrap them in a function that does two calls or something.

Reply:
Yes, it was a bit complex to understand. I changed it to loop via the number of column groups and the column groups value is controlled based on the tiered storage shared type.